lint/multitool.lock.json

```diff
@@ -35,5 +35,41 @@
         "file": "shellcheck-v0.10.0/shellcheck"
       }
     ]
+  },
+  "ruff": {
+    "binaries": [
+      {
+        "kind": "archive",
+        "os": "macos",
+        "cpu": "x86_64",
+        "url": "https://github.com/astral-sh/ruff/releases/download/v0.4.3/ruff-0.4.3-x86_64-apple-darwin.tar.gz",
```

Comment:
currently users can pick a version of ruff, I filed https://github.com/theoremlp/rules_multitool/issues/32 so we could continue to allow that while also using multitool lockfile